src/css/styles.scss

```diff
@@ -583,6 +583,20 @@
       padding: 0;
     }
   }
+
+  .excalidraw-button {
```

Comment:
let's put the CSS into a different file `button.scss` so only that CSS file is imported  when using independently and keeps it cleaner